### ISSUE TITLE
APM and Spin down time setting doesn't take effect unless reboot

### ISSUE THREAD

**Issue by weyou** [openmediavault/openmediavault#163]
Version 4.1.9

Set the APM and Spin down time and then apply the setting by the "Apply" button. The APM setting get from *hdparm -I* still keeps the previous value and the new setting doesn't work(Not see the spin down).   I checked the value in the */etc/hdparm.conf* which has been modified to the new value correctly. 

After rebooting,  everything goes okay.

**Comment by JsBergbau** [comment]
This bug seems also applies to OMV5.
Executing `sudo hdparm -S 60 /dev/sdd` works fine. However settings from the Webconfiguration are ignored.
After a reboot of the whole machine spindown works.
Is there at least a way to just restart any service that the changes are applied without rebooting the whole machine?

```
================================================================================
= openmediavault information
================================================================================
Release: 5.3.2-1
Codename: Usul
```

Until fixing there should be written a warning on that setting that you need to reboot until it is applied. It cost me at least one hour to find out whats going on and the solution.

**Comment by votdev** [comment]
Fixed in openmediavault 5.3.3.

**Comment by JsBergbau** [comment]
Just a tipp for all those who use OpenMediaVault with WD RED HDD. These disks have problems with automatic spindown. I've noticed on two systems that it works for a while, but then even hdparm -S 1 didn't work anymore (there was no load on the disk as iostat showed). hdparm -y however worked.
Even when working the HDD spun up irregulary without any real access. 

What now really works is using hd-idle, see there https://withblue.ink/2016/07/15/what-i-learnt-from-using-wd-red-disks-to-build-a-home-nas.html With that even those spin ups doen't occur anymore.
